### PR TITLE
GM Tools Name and Callsign Generation

### PR DESCRIPTION
**Pull request by Windchild292**
This adds name and callsign generation to the GM Tools dialog, handling both loading from a person and from the standard option.

As part of this I reordered and regioned the file to improve readability. I also reworked the pre-existing RAT Roller to fix a few issues I discovered it mostly relating to data load following the preferences changes.

### CODE REVIEW

File: MekHQ/resources/mekhq/resources/GMToolsDialog.properties [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# GM Tools Dialog Resource Strings
+# Dice Panel
+dicePanel.text=Dice Roller
```
Comment (sixlettervariables) [inline MekHQ/resources/mekhq/resources/GMToolsDialog.properties:3]: Weird line endings need to be resolved.

Comment (Windchild292) [reply MekHQ/resources/mekhq/resources/GMToolsDialog.properties:3]: Carriage returns, apparently. Will clean them out once I finish my current option swapover.

Comment (Windchild292) [reply MekHQ/resources/mekhq/resources/GMToolsDialog.properties]: Resolved, but had to add the Bloodname Panel start to get it to be picked up (I'm planning on handling that immediately, so no issues with it being left behind)

**Comment by Windchild292** [comment]
This has stopped working because of a casting exception and I'm not sure why it is happening now and not before. Converting to draft as I work that out.

**Review by NickAragua — APPROVED** [review]

**Pull request merged**